### DISCUSSION TITLE
How can I generate the clearest MP4 video?

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#1973] [Q&A] [answered]
I made a MP4 video, but the clarity is not enough. How can I set it to make the video the clearest? Any help would be appreciated

for exmaple, I think the text is not clear enough.

the video properties:

![image](https://user-images.githubusercontent.com/50519642/229299130-97da6c74-006f-4793-9610-0060c3663a50.png)

Here are some of my code:

// remotion.config.ts
```ts
import {Config} from 'remotion';

Config.setImageFormat('jpeg');
Config.setOverwriteOutput(true);
Config.setPort(3009);
Config.setVideoBitrate('1M');
```

// Root.tsx
```tsx
<Composition
  id="Test"
  component={Test}
  fps={60}
  durationInFrames={20 * 60}
  width={1920}
  height={1080}
/>
```

```shell
npx remotion render Test out/video.mp4
```

**Comment by JonnyBurger** [comment, marked as answer]
Instead of video bitrate, I recommend `setCrf(1)` instead.

Also, consider increasing the resolution of the video to 4k! (`--scale=2`)
Since if you are on a high-res display, 1920x1080 will not fill out all the pixels 🙂

Another solution is to render ProRes video instead, which can be lossless with the right settings!

**Reply by winner106** [reply]
thank you very much,  `--scale=2` works, and png is more clear~
